examples/Multi_Armed_Bandit/bernoulli_multi_armed_bandit.py

```diff
@@ -0,0 +1,175 @@
+## Bernoulli Multi Armed Bandit
```

Comment:
Isn't this example way too large? Didn't we once agree that examples should be rather minimal, and that lengthy explanations should be part of a user guide rather than an example?

Reply:
What exactly are you referring to? The fact that we use two different acqf functions or the "imaginary use case" part? 

Regarding the latter, I fully agree: this should eventually be moved to the user guide. But since the user guide is not yet there and Julian already drafted some text in that direction, I simply made a consistent story out of it and kept it as motivating text in the example ... for now. Once we have the user guide, we can motivate stuff there and just refer to in the examples if necessary. But I wouldn't drop the text at the moment since it's so far the only explanation on bandit models

Reply:
👍🏼